.github/workflows/ci.yml

```diff
@@ -151,12 +151,6 @@ jobs:
       - name: Lint Spell
         run: pnpm lint-spell
 
-      - name: Lint Prettier
```

Comment:
Could you keep this? We still need these to validate if autofix is executing correctly.

Reply:
yes